utils/frame/remote-externalities/src/lib.rs

```diff
@@ -379,6 +406,9 @@ impl Builder {
 #[cfg(test)]
 mod tests {
 	use super::*;
+	pub type Header = sp_runtime::generic::Header<u32, sp_runtime::traits::BlakeTwo256>;
```

Comment:
You could also just use these: https://crates.parity.io/sp_runtime/testing/index.html

Reply:
I replaced them with the types in sp_runtime::testing